src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -100,6 +109,17 @@ public HostStatus GetHostStatus()
             return status;
         }
 
+        [HttpPost]
+        [Route("admin/host/debug")]
+        public bool LaunchDebugger()
+        {
+            if (_webHostSettings.IsSelfHost)
+            {
+                return Debugger.Launch();
```

Comment:
Is this idempotent? E.g. what happens if you call this 5 times in a row? Should you use Debugger.IsAttached as a guard, or is it ok to call repeatedly?